packages/devtools_app/lib/src/scaffold.dart

```diff
@@ -37,7 +37,7 @@ class DevToolsScaffold extends StatefulWidget {
     Key key,
     @required this.tabs,
     @required this.analyticsProvider,
-    this.title,
+    this.title = 'DevTools for Flutter & Dart',
```

Comment:
this is a little of an ugly default value. Are multiple different screens using it?

Reply:
The landing screen was already using it. The snapshot screen (for offline imports) and app size screen had no title, so adding this makes them the same as the landing screen.